tests/CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ target_compile_options(gsl_tests_config INTERFACE
         -Werror
         -Wextra
         -Wno-missing-braces
+        -Wno-unknown-attributes
```

Comment:
I bet you have tested it, but I cannot find any information online describing what `-Wunknown-attributes` or `-Wno-unknown-attributes` do. And confusingly enough I find code that uses `-Wunknown-attribute` (no 's'). Do you have a link to a page describing it (gcc?, clang?)?

Reply:
I couldn't find any documentation but clang gives  the following warnings on unknown attrobutes:

> warning: unknown attribute ... ignored [-Wunknown-attributes]

So I took a guess at removing that:)

Reply:
Can't reply to your comment below - sounds reasonable, created PR #719